theseus/labs/lie_functional/__init__.py

```diff
@@ -2,3 +2,56 @@
 #
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
+from typing import Callable, Optional, Sequence
+
+import torch
+
+import theseus.labs.lie_functional.se3 as _se3_impl
+import theseus.labs.lie_functional.so3 as _so3_impl
+from .constants import DeviceType
+from .lie_group import BinaryOperatorFactory, UnaryOperatorFactory
+
+_CheckFnType = Callable[[torch.Tensor], None]
+_RadnFnType = Callable[
+    [
+        Sequence[int],
+        Optional[torch.Generator],
+        Optional[torch.dtype],
+        DeviceType,
+        bool,
+    ],
+    torch.Tensor,
+]
+
+
+# Namespace to facilitate type-checking downstream
+class LieGroupFns:
+    def __init__(self, module):
+        self.exp, self.jexp = UnaryOperatorFactory(module, "exp")
```

Comment:
These functions get called internally after each op is defined in for example se3.py. Could we just instead call this class at the end of each file? Do we even need to call it if these are already being called below in this file?

Reply:
Ah yes, sorry, my plan was to remove the original function calls, I forgot about this. Thanks for pointing it out!